wazo_lib_rest_client/command.py

```diff
@@ -37,3 +39,10 @@ def __init__(self, client):
         super(RESTCommand, self).__init__(client)
         self.base_url = self._client.url(self.resource)
         self.timeout = self._client.timeout
+
+    def _get_headers(self, **kwargs):
```

Comment:
Why not `_get_ro_headers`? that way we could then also add `_rw_headers` and `_get_rw_headers`?